packages/app/src/App.tsx

```diff
@@ -40,6 +40,7 @@ import { Navigate, Route } from 'react-router';
 import { apis } from './apis';
 import { EntityPage } from './components/catalog/EntityPage';
 import Root from './components/Root';
+import { SearchPage } from './components/search/SearchPage';
```

Comment:
The thinking is that, while we want to provide a reasonable default search UI (with some facets, a search input, and a list of results), people will want to customize the heck out of it.  And from that POV, we'd just provide a good starting point in the app (here for now, a copy in `create-app` when it's ready).

Reply:
App would get real messy if every plugin took this approach though, and it would be almost impossible to keep a created app up to date with changes.

I think we want to ship a decent built-in search page that covers 50% of all use-cases, then we might build in some extension points that allow another 40% to customize it to their needs using some extra code, and the last 10% simply creates their own search page plugin that they use instead of the built-in one.

Reply:
We're thinking Search might be a special case, here.  It's easy to imagine...

```jsx
<SearchResults>
  {results.map(result => {
    switch (result.index) {
      case: 'software-catalog':
        return <CatalogEntityResult document={result} />
      case: 'techdocs':
        return <TechDocResult document={result} />
      default:
        return <DefaultResult document={result} />
    }
  })}
</SearchResults>
```

Which works out of the box for basically everyone and would probably change very infrequently.

...But then you create/install an `Experimentation Platform` plugin that exposes A/B tests as searchable things, and it would be really useful to display whether or not that experiment is currently active in the list of search results.

We're thinking it'd be a better experience in that situation to just go to a page that already exists in the created app, rather than having to create one, copy / paste from the existing plugin, and then add the result component.